src/services/log/Log_service.cc

```diff
@@ -16,6 +16,10 @@ Log_service::Log_service(JApplication *app) {
     m_log_level_str = "info";
     m_application->SetDefaultParameter("eicrecon:LogLevel", m_log_level_str, "log_level: trace, debug, info, warn, error, critical, off");
     spdlog::default_logger()->set_level(eicrecon::ParseLogLevel(m_log_level_str));
+
+    m_log_format_str = "[%n] [%^%l%$] %v";
+    m_application->SetDefaultParameter("eicrecon:LogFormat", m_log_level_str, "spdlog pattern string");
```

Comment:
Wasn't there something about spaces that `SetDefaultParameter` couldn't handle in the adjacency matrix?

Reply:
Good point. I don't see an issue in CI, is that because the default value is unaffected?

Reply:
Yeah, I think the default is not affected. The `SetDefaultParameter` function doesn't set the default parameter that is passed as the argument, but instead it overwrites the variable (passed by reference) with the non-default value passed on the command line... Maybe `GetNonDefaultParameter` would have been clearer.

Reply:
Who knows what other characters JANA2 might trip up over (I like my logging like valgrind... `eicrecon -Peicrecon:LogFormat="==%n==\\%10!!"`) but I'd say let's get this in, clean up the logs, and we can flag this again upstream.